server/broadcast.go

```diff
@@ -563,7 +564,7 @@ func (bsm *BroadcastSessionsManager) chooseResults(ctx context.Context, submitRe
 		} else {
 			sessionsToSuspend = append(sessionsToSuspend, untrustedResult.Session)
 			if monitor.Enabled {
-				monitor.FastVerificationFailed(ctx)
+				monitor.FastVerificationFailed(ctx, ouri)
```

Comment:
How does this URI look? Do you have an example of the metric that's emitted?

Just thinking about if we want to normalize it into something a little less verbose than a full URI

Reply:
> How does this URI look? Do you have an example of the metric that's emitted?

I referred the following [comment ](https://github.com/livepeer/go-livepeer/issues/2371#issuecomment-1121715955)and [here](https://github.com/livepeer/go-livepeer/blob/4d80d57efbb99e57317d9ce5722101def44ef48e/monitor/census.go#L1135).

> Just thinking about if we want to normalize it into something a little less verbose than a full URI

I think we have to consider @leszko's opinion [here](https://github.com/livepeer/go-livepeer/issues/2371#issuecomment-1124671741).